samples/StaticHtmlComposites/package.json

```diff
@@ -7,6 +7,8 @@
     "build": "webpack --mode=production",
     "start": "concurrently \"cd ../Server && rushx start:dev\" \"rushx start:app\"",
     "start:app": "webpack serve",
+    "test": "playwright test --workers=1 tests",
```

Comment:
Can `--workers=1` go in the playright config? It would be great if we can keep configuration params in one place